apollo-runtime/src/jvmMain/kotlin/com/apollographql/apollo3/network/ws/OkHttpWebSocketEngine.kt

```diff
@@ -105,4 +108,9 @@ actual class DefaultWebSocketEngine(
       }
     }
   }
+
+  @Deprecated("Use open(String, List<HttpHeader>) instead.", ReplaceWith("open(String, List<HttpHeader>)"))
```

Comment:
```suggestion
  @Deprecated("Use open(String, List<HttpHeader>) instead.", ReplaceWith("open(url, headers)"))
```

I think the Kotlin intelliJ plugin is smart enough to replace the arguments values

Reply:
@sproctor want me to look into this?